### PR TITLE
fix(submit): add transparency on CL operations

### PR DESCRIPTION
**Pull request by k-dovgan**

### CODE REVIEW

File: universum/modules/vcs/perforce_vcs.py [hunk 1]
```diff
@@ -199,6 +199,7 @@ def submit_new_change(self, description, file_list, review=False, edit_only=Fals
         change["Files"] = []
         change["Description"] = description
         change_id = self.p4.save_change(change)[0].split()[1]
+        self.out.log(f"Created empty change {change_id} for reconciling.")
```
Comment (i-keliukh) [inline universum/modules/vcs/perforce_vcs.py:202]: Is "change" the correct term used by the perforce?

File: universum/modules/vcs/perforce_vcs.py [hunk 2]
```diff
@@ -208,12 +209,14 @@ def submit_new_change(self, description, file_list, review=False, edit_only=Fals
             # If no changes were reconciled, there will be no file records in CL dictionary
             if "Files" not in current_cl:
                 self.p4.run_change("-d", change_id)
+                self.out.log(f"Deleted empty CL {change_id}.")
```
Comment (i-keliukh) [inline universum/modules/vcs/perforce_vcs.py:212]: Previously you used the term "change", but now you use "CL".

File: universum/modules/vcs/perforce_vcs.py (hunk 1 above)
Comment (i-keliukh) [reply universum/modules/vcs/perforce_vcs.py:202]: I prefer "CL"

**Comment by i-keliukh** [comment]
Could you please post the log example to PR comments?

File: universum/modules/vcs/perforce_vcs.py (hunk 1 above)
Comment (k-dovgan) [reply universum/modules/vcs/perforce_vcs.py:202]: I think I've seen term 'change' somewhere in official p4 docs, and therefore tend to use them as synonyms. But 'CL' is definitely more formal.

**Comment by k-dovgan** [comment]
> Could you please post the log example to PR comments?

Here's an example of a successful submit (most common usecase):
```
==> Universum 0.19.9 started execution
1. Executing
 |   1.1. Connecting
 |      |   1.1.1. Disconnecting
 |      |        └ [Success]
 |      |   
 |      └ [Success]
 |   
 |   ==> Created empty change 11 for reconciling.
 |   ==> Change 11 submitted
 └ [Success]

2. Finalizing
 |   2.1. Finalizing
 |      |   2.1.1. Disconnecting
 |      |        └ [Success]
 |      |   
 |      └ [Success]
 |   
 └ [Success]

==> Universum 0.19.9 finished execution
```

**Comment by k-dovgan** [comment]
Also here's an example of a failed submit:
```
==> Universum 0.19.9 started execution
1. Executing
 |   1.1. Connecting
 |      |   1.1.1. Disconnecting
 |      |        └ [Success]
 |      |   
 |      └ [Success]
 |   
 |   ==> Created empty change 13 for reconciling.
 |   ==> Reverted and deleted CL 13
 |   Error: [P4#run] Errors during command execution( "p4 submit -f revertunchanged -i" )
 |   
 |   	[Error]: "No files to submit.\nFile(s) couldn't be locked.\nSubmit failed -- fix problems above then use 'p4 submit -c 13'."
 |   
 └ [Failed]

2. Finalizing
 |   2.1. Finalizing
 |      |   2.1.1. Disconnecting
 |      |        └ [Success]
 |      |   
 |      └ [Success]
 |   
 └ [Success]

==> Universum 0.19.9 finished execution
```

**Comment by i-keliukh** [comment]
Should we mention files in the CL as well?

**Comment by k-dovgan** [comment]
> Should we mention files in the CL as well?

Actually that's a very good idea.

**Review by i-keliukh — APPROVED** [review]

**Pull request merged**